internal/pubsub/pubsub.go

```diff
@@ -233,14 +220,12 @@ func NewSubscription(d driver.Subscription) *Subscription {
 		}
 		// TODO: Consider providing a way to stop this call. See #766.
 		callCtx := context.Background()
-		err := retry.Call(callCtx, gax.Backoff{}, d.IsRetryable, func() error {
+		return retry.Call(callCtx, gax.Backoff{}, d.IsRetryable, func() error {
 			return d.SendAcks(callCtx, ids)
 		})
-		// TODO(#695): Do something sensible if SendAcks returns an error.
-		_ = err
 	}
-	ab := bundler.NewBundler(ackIDBox{}, handler)
-	ab.DelayThreshold = time.Millisecond
+	maxHandlers := 1
+	ab := batcher.New(ackIDBox{}, maxHandlers, handler)
```

Comment:
ackIDBox is no longer necessary. The first arg should be `reflect.TypeOf([]driver.AckID{}).Elem()`

Reply:
Done.

Reply:
Filed https://github.com/google/go-cloud/issues/814